tasks/eva_3399/src/main/groovy/eva3399/eva3380_collect_colliding_ss_hashes.groovy

```diff
@@ -34,68 +33,22 @@ if (!options) {
 }
 def logger = LoggerFactory.getLogger(this.class)
 
-// Copied verbatim from here: https://github.com/EBIvariation/eva-tasks/blob/27e4a1bda59fb30e8baabd2500119ae9ffacf05d/tasks/eva_3371/src/main/groovy/eva3371/eva3371_detect_unnormalized_indels.groovy#L48
-def runProcess = {String command ->
-    logger.info("Running command: ${command}...")
-    def process= ["bash", "-c", command].execute()
-    def (out, err) = [new StringBuffer(), new StringBuffer()]
-    process.waitForProcessOutput(out, err)
-    if(!out.toString().trim().equals("")) logger.info(out.toString())
-    if (process.exitValue() != 0) {
-        logger.error("Command: $command exited with exit code ${process.exitValue()}!!")
-        logger.error("See error messages below: \n" + err.toString())
-        throw new Exception(err.toString())
-    }
-    logger.warn(err.toString())
-    logger.info("Command: $command completed successfully!")
-    return ["out": out.toString(), "err": err.toString()]
-}
-
-def getCustomAssemblyReportPath = {String assembly, String fastaDir ->
+static String getCustomAssemblyReportPath (String assembly, String fastaDir) {
     def customAssemblyReportPath = runProcess("find ${fastaDir} -maxdepth 3 -iname ".toString() +
             "'${assembly}_assembly_report_custom.txt'| head -1".toString()).out.trim()
     if (customAssemblyReportPath.equals("")) throw new Exception("Could not find assembly report for assembly $assembly!!")
     return customAssemblyReportPath
 }
 
-// Copied verbatim from here: https://github.com/EBIvariation/eva-tasks/blob/27e4a1bda59fb30e8baabd2500119ae9ffacf05d/tasks/eva_3371/src/main/groovy/eva3371/eva3371_detect_unnormalized_indels.groovy#L154
-def getNormalizedDbsnpAndEvaSves = { ArrayList<Variant> variantsPostNorm,
```

Comment:
Method now only in [eva3371_detect_unnormalized_indels.groovy](https://github.com/EBIvariation/eva-tasks/pull/144/files#diff-b783091d22d24ea4bb2c146bf9c473582d9eb7baaab6e9cfc4f32163f55def7f)